### PR TITLE
Update venues.js

### PR DESCRIPTION
**Pull request by Connorsmith25**
changed schema for venues to be geojson format

### CODE REVIEW

**Review by ztylergeorge — COMMENTED** [review]
I think it looks fine, just a comment/question -- is the Python script affected by this change? I don't think it should since the information is basically just going to be nested inside of properties, but looking for other thoughts.

File: taps-api/data/venues.js [hunk 1]
```diff
@@ -28,19 +28,22 @@ module.exports = {
...
-      _id: uuid.v4(),
-      name,
-      city,
-      state,
-      country,
-      lat,
-      lng,
-      beersAvailable: []
+      type: "Point",
+      coordinates: [lng, lat],
```
Comment (alexsaltstein) [inline taps-api/data/venues.js:34]: One thing that needs to be changed here is that lng and lat are both strings atm. We need to use a parse function to make these numbers.

File: taps-api/data/venues.js [hunk 2]
```diff
@@ -28,19 +28,22 @@ module.exports = {
     if (typeof lng !== 'string' || lng.trim() === '') throw new Error('lng must be a non-empty string');
```
Comment (alexsaltstein) [inline taps-api/data/venues.js:28]: When parsing numbers make sure they are between the valid coordinates that they can be according to mongo website.

Comment (alexsaltstein) [reply taps-api/data/venues.js:28]: https://docs.mongodb.com/manual/reference/geojson/

**Pull request merged**